airflow-core/src/airflow/ui/public/i18n/locales/de/dashboard.json

```
@@ -1,4 +1,10 @@
 {
+  "favorite": {
+    "favoriteDags_one": "Erstes favorisiertes DAG",
+    "favoriteDags_other": "Top {{count}} favorisierte DAGs",
+    "noDagRuns": "Noch kein Lauf für das Dag.",
+    "noFavoriteDags": "Keine Favoriten. Mit dem Stern-Symbol neben einem Dag kann man ein Dag in diese Liste hinzufügen."
```

```suggestion
    "noFavoriteDags": "Noch keine favorisierten Dags. Mit dem Stern-Symbol neben einem Dag kann man das Dag zu den Favoriten hinzufügen."
```

Clarification suggestion :)